RELEASING.md

```diff
@@ -0,0 +1,61 @@
+# Releasing
```

Comment:
@secondsun @wtrocki I have added an entire section on releasing independent sub modules. Obviously this method has not been executed yet, so we may find some things that do not work when releasing the next time. Worth to have a read.